test/e2e/run.sh

```diff
@@ -290,6 +299,48 @@ run-hook() {
     eval "${hook_code}"
 }
 
+install-files() {
```

Comment:
nitpick: wouldn't it be more consistent to call this `install{-_}vm{-_}files`, considering how the related/similar `export_vm_files()` and `export_var_files()`  functions and the `vm-files` variable itself called ?

bikeshed: We seem to have two naming conventions here for multipart function names. One is using underscores (`export_vm_files`), and another using hyphens (`install-files`). Is there a particular logic to this or is it related to something (eg. temporal correlation with the demo->e2e-tests evolution timeline) ?

Reply:
trying to keep using `-` wherever possible to keep function naming consistent. Renamed `export_{vm,var}_files` to `export-{vm,var}-files`.